example/src/Examples/API/Picture.tsx

```diff
@@ -47,3 +52,31 @@ const styles = StyleSheet.create({
     flex: 1,
   },
 });
+
+interface RecordPictureProps {
+  onRecord: (picture: SkPicture) => void;
+  bounds: SkRect;
+  children?: ReactNode | ReactNode[];
+}
+
+const RecordPicture = ({ onRecord, bounds, ...props }: RecordPictureProps) => {
+  const [picture, setPicture] = useState<null | SkPicture>(null);
```

Comment:
I thought we would just record the picture once and if people need to do many recordings they have control over it via the `key` prop we could make this more explicit of course.